src/styles/mailchimp.scss

```diff
@@ -7,35 +7,29 @@ $mailchimp-color-white: #fff;
 #mc_embed_signup input[type="text"] {
   font-size: 14px;
   font-weight: 400;
-  line-height: 27px;
+  line-height: 32px;
   color: $mailchimp-color-grey;
   outline: 0;
   border-width: 2px;
-  width: 400px;
-  padding-top: 7px; /* aligns input text in element */
+  width: 390px;
+  /*padding-top: 7px;  aligns input text in element */
   background-color: $mailchimp-color-white;
   border-radius: 30px;
-  text-align: center;
+  text-align: left;
   display: inline-block;
   text-transform: none;
   border-radius: 30px;
   border: 2px solid $mailchimp-color-main;
   transition: all 0.15s ease-in-out;
-
+  font-family: "Brown";
+  padding-left: 10px;
```

Comment:
`padding-left` shifts the submit button on smaller screen sizes 

![image](https://user-images.githubusercontent.com/23108291/68311566-b6721d80-007f-11ea-9fcd-17a772160650.png)

Reply:
I think I fixed it by doing box-sizing: border-box; and then playing around with the styles, hopefully it's good as a temporary fix